examples/helloworld.rs

```diff
@@ -0,0 +1,53 @@
+use std::{
+    env,
+    ffi::{c_void, CString},
+    os::raw::c_char,
+    ptr,
+};
+
+use dpdk_sys::{rte_eal_remote_launch, rte_get_next_lcore, RTE_MAX_LCORE};
+
+unsafe extern "C" fn lcore_main(_args: *mut c_void) -> i32 {
+    println!("hello from lcore");
+    0
+}
+
+fn main() {
```

Comment:
I recommend to translate the C version [hello world](https://github.com/DPDK/dpdk/blob/main/examples/helloworld/main.c) example to the Rust version here.

Reply:
> I recommend to translate the C version [hello world](https://github.com/DPDK/dpdk/blob/main/examples/helloworld/main.c) example to the Rust version here.

Now the `lcore_main` function is the same as DPDK's. (However, to use the `#[thread_local]` attribute, the Rust version became nightly...)